### PR TITLE
Some minor code improvements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Comment by rafaelfranca** [comment]
Thank you for the pull request. We don't return singleton `true`/`false` in methods ending with `?` since in Ruby anything that is not `nil` or `false` is true.

File: lib/thor/shell/basic.rb [hunk 1]
```diff
@@ -276,14 +276,15 @@ def file_collision(destination)
       # This code was copied from Rake, available under MIT-LICENSE
       # Copyright (c) 2003, 2004 Jim Weirich
       def terminal_width
+        DEFAULT_WIDTH = 80
```
Comment (rafaelfranca) [inline lib/thor/shell/basic.rb]: To this constant make sense it can't be inside the method.

**Comment by sschuberth** [comment]
> We don't return singleton `true`/`false` in methods ending with `?` since in Ruby anything that is not `nil` or `false` is true.

While this is syntactically correct, this is considered bad practice, see e.g. https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark. Also, IIRC I found this because it was flagged by some static code analyzer (I don't remember which one).

File: lib/thor/shell/basic.rb (hunk 1 above)
Comment (sschuberth) [inline lib/thor/shell/basic.rb]: Why, if it's not used elsewhere (yet)?

**Comment by rafaelfranca** [comment]
> While this is syntactically correct, this is considered bad practice

What is bad practice or not is defined by the project maintainer. We don't follow the bbatsov's ruby-stile-guide in that rule, neither Rails does, so could you remove that commit?

**Comment by sschuberth** [comment]
> What is bad practice or not is defined by the project maintainer.

Yeah, in a world of alternative facts that's probably true :wink:

> We don't follow the bbatsov's ruby-stile-guide in that rule, neither Rails does, so could you remove that commit?

Done, anyway.

**Comment by rafaelfranca** [comment]
Thank you so much for doing those kinds of improvements. Really appreciated

**Pull request merged**